src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  min-width: 1024px;
+  display: flex;
+  background-color: white;
+}
+
+.logo {
+  margin: auto;
+  margin-left: 50px;
+}
+
+.nav {
+  margin: auto;
+  width: 100%;
+  margin-right: 50px;
+}
```

Comment:
The 'nav' class has the same issue with margins as the 'logo' class. Please use shorthand notation for margins.